SubmitReview/ReadMe.md

```diff
@@ -2,8 +2,8 @@
 
 Move fixed Jira tickets using the 'Request Changes' or 'Approved' transition after submitting a review on a pull request.
 
-If the pull request has been approved, the `Approved` transition will be used, moving, typically, it would a ticket from`IN REVIEW` to `APPROVED`.
-If changes are requested, the `Request Changes` transition will be used, moving, typically, it would a ticket from`IN REVIEW` to `IN PROGRESS`.
+If the pull request has been approved, the `Approved` transition will be used, it would typically move a ticket from `IN REVIEW` to `APPROVED`.
```

Comment:
Styling inconsistency
```suggestion
If the pull request has been approved, the `Approved` transition will be used. It would typically move a ticket from `IN REVIEW` to `APPROVED`.
```